src/bin/add/args.rs

```diff
@@ -117,14 +111,9 @@ pub struct Args {
     #[structopt(long = "allow-prerelease")]
     pub allow_prerelease: bool,
 
-    /// Space-separated list of features to add. For an alternative approach to
-    /// enabling features, consider installing the `cargo-feature` utility.
-    #[structopt(long = "features", number_of_values = 1)]
-    pub features: Option<Vec<String>>,
-
-    /// Set `default-features = false` for the added dependency.
-    #[structopt(long = "no-default-features")]
-    pub no_default_features: bool,
+    /// For an alternative approach to enabling features, consider installing `cargo-feature`.
+    #[structopt(flatten)]
+    pub features: clap_cargo::Features,
```

Comment:
This added an `all_features` that is going unused.  I think that will be confusing to users.